tests/test_wave_01.py

```diff
@@ -131,13 +132,13 @@ def test_update_task_not_found(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
+    # raise Exception("Complete test with assertion about response body")
     # *****************************************************************
     # **Complete test with assertion about response body***************
     # *****************************************************************
+    assert response_body == {"message" : "Task 1 is not found"}
```

Comment:
Same comment as above, this message you have written in the assertion is slightly off from what the test expects.